core/services/functions/listener.go

```diff
@@ -417,6 +426,17 @@ func (l *FunctionsListener) handleOracleRequest(request *ocr2dr_oracle.OCR2DROra
 		return
 	}
 
+	reportedDomainsJson, errDomains := l.jobORM.FindTaskResultByRunIDAndTaskName(run.ID, ParseDomainsTaskName, pg.WithParentCtx(ctx))
+	if errDomains == nil && reportedDomainsJson != nil && len(reportedDomainsJson) > 0 {
+		var reportedDomains []string
+		errJson := json.Unmarshal(reportedDomainsJson, &reportedDomains)
+		if errJson != nil {
+			l.logger.Warnw("failed to parse reported domains", "requestID", formatRequestId(request.RequestId), "err", errJson)
+		} else if len(reportedDomains) > 0 {
+			l.reportSourceCodeDomains(request.RequestId, reportedDomains)
+		}
+	}
```

Comment:
You could log non-nil errDomains as that would indicate some DB problems (missing entry?)

Reply:
Good idea, done.